jane/doc/extensions/modes/syntax.md

```diff
@@ -87,6 +87,21 @@ val foo : string -> string @ local @@ portable
 (expression : _ @@ local)
 ```
 
+## Modules
+```ocaml
+module M : S @@ portable = ..
```

Comment:
But these syntaxes aren't really supported yet, right? That is, these parse, but they don't work yet, if I understand correctly. I don't think these should be in the user-facing documentation before they work.

Reply:
Ugh good point. I will remove.